components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java

```diff
@@ -79,6 +79,40 @@
         return groups;
     }
 
+    /**
+     * Lists the groups that belong to a tenant by its tenant id that are created from SCIM
+     *
+     * @param tenantId tenant Id
+     * @return The set of groups that were created from SCIM
+     * @throws IdentitySCIMException Error when reading from persistence store.
+     */
+    public Set<String> listSCIMGroups(int tenantId) throws IdentitySCIMException {
+        Connection connection = IdentityDatabaseUtil.getDBConnection();
+        PreparedStatement prepStmt = null;
+        ResultSet resultSet = null;
+        Set<String> groups = new HashSet<>();
+
+        try {
+            //retrieve groups from the DB
```

Comment:
`.` at the end of the comment and start the comment with an uppercase letter.

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/252/commits/98c7dd4b02908f8feb007d39d8f7fca4a7abae6d